### PR TITLE
Users table from nav bar

### PR DESCRIPTION
**Pull request by juliasaveliff**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -63,18 +61,14 @@ class App extends React.Component<AppProps> {
...
-              component={AssetView}
-            />
+            <this.PrivateRoute path="/assets/:rid" component={AssetView} />
             <this.PrivateRoute path="/report" component={Report} />
             {/* admin paths */}
             <this.PrivateRoute
-              path="/register"
-              component={
-                this.props.isAdmin ? WrappedNormalRegistrationForm : Notfound
-              }
```
Comment (clairecfu) [inline src/App.tsx]: removing the create user button from nav bar

File: src/components/elementView/elementTab.tsx [hunk 2]
```diff
@@ -148,31 +157,66 @@ class ElementTab extends React.Component<ElementTabProps, ElementViewState> {
...
+  private addErrorToast(message: string) {
+    this.addToast({ message: message, intent: Intent.DANGER });
+  }
+  private addToast(toast: IToastProps) {
+    toast.timeout = 5000;
+    this.toaster.show(toast);
+  }
+  private toaster: Toaster = {} as Toaster;
+  private refHandlers = {
+    toaster: (ref: Toaster) => (this.toaster = ref)
```
Comment (clairecfu) [inline src/components/elementView/elementTab.tsx]: adding logic for toasts

File: src/components/elementView/elementTab.tsx [hunk 3]
```diff
@@ -255,6 +301,8 @@ class ElementTab extends React.Component<ElementTabProps, ElementViewState> {
               this.setState({ filters: data });
             }}
             shouldUpdateData={this.state.updateTable}
+            disableSorting={this.props.element === ElementType.USER}
+            disableFiltering={this.props.element === ElementType.USER}
```
Comment (clairecfu) [inline src/components/elementView/elementTab.tsx]: disable sorting/filtering for User tab

File: src/forms/auth/register.tsx [hunk 4]
```diff
@@ -213,8 +194,15 @@ class RegistrationForm extends React.Component<
...
-const WrappedRegistrationForm = Form.create()(RegistrationForm);
+const WrappedRegistrationForm = Form.create()(withRouter(RegistrationForm));
 
+function addProps(RegForm: any) {
+  return class extends React.Component<RegistrationFormSubmitProps> {
+    render() {
+      return <RegForm {...this.props} />;
+    }
+  };
+}
```
Comment (clairecfu) [inline src/forms/auth/register.tsx]: had to add this function thingy to allow props to be passed into the WrappedRegistrationFormComponent

Comment (clairecfu) [reply src/forms/auth/register.tsx]: janky but it works!

File: src/components/elementView/elementTab.tsx (hunk 3 above)
Comment (juliasaveliff) [reply src/components/elementView/elementTab.tsx]: groovy

**Pull request merged**